rules/windows/process_creation/win_susp_use_of_sqlps_bin.yml

```diff
@@ -0,0 +1,26 @@
+title: Detection of PowerShell Execution via SQL 
+id: 0152550d-3a26-4efd-9f0e-54a0b28ae2f3  
+status: experimental
+description: PowerShell execution through builtin SQL Server "SQLPS.exe" binary.
```

Comment:
Could you also please mention that the problem with that binary is that the commands are not going to be logged into the PowerShell logs? it's quite an important moment. Thank you!